lib/model/dbConn.js

```diff
@@ -190,10 +190,7 @@ function configureDb(callback) {
     /*jshint camelcase:false, validthis:true */
     const currentConfig = config.getConfig();
 
-    if (
-        (currentConfig.deviceRegistry && currentConfig.deviceRegistry.type === 'mongodb') ||
-        (currentConfig.stats && currentConfig.stats.persistence === true)
-    ) {
+    if (currentConfig.deviceRegistry && currentConfig.deviceRegistry.type === 'mongodb') {
```

Comment:
https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/admin.md#stats section in documentation should be removed? Or it is still used?

Reply:
it has been removed, no longer used.

Reply:
Ok. I missed that... NTC